extensions/amp-story/1.0/amp-story.js

```diff
@@ -418,6 +424,20 @@ export class AmpStory extends AMP.BaseElement {
     });
   }
 
+  /**
+  * @private
+  */
+  setThemeColor_() {
+    // The theme color should be copied from the story's primary accent color
+    // if possible, with the fall back being default light gray.
+    let meta = document.createElement('meta');
+    const styles = computedStyle(this.win.document.body);
+    meta.name = 'theme-color';
+    meta.content = styles.getPropertyValue('primary-color') ||
+                   DEFAULT_THEME_COLOR;
```

Comment:
nit: I think the style is to indent hanging lines +4 spaces from the previous line.  So:

```javascript
meta.content = styles.getPropertyValue('primary-color') ||
    DEFAULT_THEME_COLOR;
```